testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js

```diff
@@ -1430,42 +1430,42 @@ QUnit.module('Grouping', () => {
         const items = form._testResultItems;
 
         items[0].template.render({
-            model: {},
+            model: { editorOptions: { inputAttr: {} } },
             container: template
```

Comment:
Why do we need all of this changed in tests?
Shouldn't we use optional chaining in the code https://github.com/DevExpress/DevExtreme/pull/24521/files#diff-aa52d76cd61878684b8de01d6631bccd4cc5ec63a16e0981048f6382a33741e6R529?

Reply:
editorOptions is always exist and optional chaining has not any sense in real world. This test run render method with empty model, but in the reality model is not empty